source/includes/steps-starting-compass.yaml

```diff
@@ -60,9 +67,12 @@ content: |
    the :guilabel:`Connect` button to navigate to the |compass-short|
    :doc:`Home Page </instance>`.
```

Comment:
In the note about the system-specific credentials API, that first sentence is quite dense at the end, perhaps try:

> MongoDB Compass securely stores sensitive information enter in the connecting form using a credentials API that is specific to your operating system.

Is that what that sentence is intended to say?

Reply:
Looks like we changed it to:

> MongoDB Compass securely stores sensitive information entered in the connection form using an API that is specific to your operating system API.

Should it be "system's" in the second to last word?

Reply:
Oops had an extra word. Fixed.

Reply:
Looks good!